sopel/config/__init__.py

```diff
@@ -159,6 +159,31 @@ def homedir(self):
         else:
             return os.path.dirname(self.filename)
 
+    def get_defined_sections(self):
+        """Retrieve all defined static section of this configuration.
+
+        :return: a list of all instance of
+                 :class:`~sopel.config.types.StaticSection` defined for this
+                 configuration file.
+        :rtype: list
```

Comment:
nemesi**S** and elimination of redundant info

```suggestion
        :return: all instances of :class:`~sopel.config.types.StaticSection`
                 defined for this configuration file
        :rtype: list
```

The only downside to this change is the widow it introduces in our current doc styles (note "file" on its own line):

![image](https://user-images.githubusercontent.com/164140/97721663-b8ef3380-1a97-11eb-8bde-bf5c9c48534d.png)